projects/1.16/assets/techreborn/techreborn/lang/zh_cn.json

```
@@ -1 +1,984 @@
+  "item.techreborn.debug_tool": "调试工具",
+  "item.techreborn.industrial_chainsaw": "工业电锯",
+  "item.techreborn.industrial_drill": "工业钻头",
+  "item.techreborn.industrial_jackhammer": "工业凿岩机",
+  "item.techreborn.electric_treetap": "电动树脂提取器",
+  "item.techreborn.nanosaber": "纳米剑",
+  "item.techreborn.omni_tool": "万能工具",
+  "item.techreborn.rock_cutter": "岩石切割机",
+  "item.techreborn.basic_chainsaw": "基础电锯",
+  "item.techreborn.basic_drill": "基础钻头",
+  "item.techreborn.basic_jackhammer": "基础凿岩锤",
+  "item.techreborn.treetap": "木龙头",
+  "item.techreborn.wrench": "扳手",
+  "item.techreborn.destructopack": "解构包",
+  "item.techreborn.painting_tool": "喷漆工具",
+  "_comment17": "物品-杂项",
+  "item.techreborn.cell": "空单元",
+  "item.techreborn.cell.fluid": "$fluid$单元",
+  "item.techreborn.frequency_transmitter": "频率转换器",
+  "item.techreborn.manual": "科技手册",
+  "item.techreborn.missingRecipe": "未知物品",
+  "item.techreborn.scrap_box": "废料盒",
+  "item.techreborn.overclocker_upgrade": "超频升级",
+  "item.techreborn.transformer_upgrade": "高压升级",
+  "item.techreborn.energy_storage_upgrade": "储能升级",
+  "item.techreborn.superconductor_upgrade": "超导升级",
+  "item.techreborn.bronze_sword": "青铜剑",
+  "item.techreborn.bronze_pickaxe": "青铜镐",
+  "item.techreborn.bronze_spade": "青铜铲",
+  "item.techreborn.bronze_axe": "青铜斧",
+  "item.techreborn.bronze_hoe": "青铜锄",
+  "item.techreborn.bronze_helmet": "青铜头盔",
+  "item.techreborn.bronze_chestplate": "青铜胸甲",
+  "item.techreborn.bronze_leggings": "青铜护腿",
+  "item.techreborn.bronze_boots": "青铜靴子",
+  "item.techreborn.ruby_sword": "红宝石剑",
+  "item.techreborn.ruby_pickaxe": "红宝石镐",
+  "item.techreborn.ruby_spade": "红宝石铲",
+  "item.techreborn.ruby_axe": "红宝石斧",
+  "item.techreborn.ruby_hoe": "红宝石锄",
+  "item.techreborn.ruby_helmet": "红宝石头盔",
+  "item.techreborn.ruby_chestplate": "红宝石胸甲",
+  "item.techreborn.ruby_leggings": "红宝石护腿",
+  "item.techreborn.ruby_boots": "红宝石靴子",
+  "item.techreborn.sapphire_sword": "蓝宝石剑",
+  "item.techreborn.sapphire_pickaxe": "蓝宝石镐",
+  "item.techreborn.sapphire_spade": "蓝宝石铲",
+  "item.techreborn.sapphire_axe": "蓝宝石斧",
+  "item.techreborn.sapphire_hoe": "蓝宝石锄",
+  "item.techreborn.sapphire_helmet": "蓝宝石头盔",
+  "item.techreborn.sapphire_chestplate": "蓝宝石胸甲",
+  "item.techreborn.sapphire_leggings": "蓝宝石护腿",
+  "item.techreborn.sapphire_boots": "蓝宝石靴子",
+  "item.techreborn.peridot_sword": "橄榄石剑",
+  "item.techreborn.peridot_pickaxe": "橄榄石镐",
+  "item.techreborn.peridot_spade": "橄榄石铲",
+  "item.techreborn.peridot_axe": "橄榄石斧",
+  "item.techreborn.peridot_hoe": "橄榄石锄",
+  "item.techreborn.peridot_helmet": "橄榄石头盔",
+  "item.techreborn.peridot_chestplate": "橄榄石胸甲",
+  "item.techreborn.peridot_leggings": "橄榄石护腿",
+  "item.techreborn.peridot_boots": "橄榄石靴子",
+  "_comment18": "信息",
+  "techreborn.message.missingmultiblock": "多方块结构缺失",
+  "techreborn.message.setTo": "设置为",
+  "techreborn.message.in": "在",
+  "techreborn.message.noCoordsSet": "没有设置坐标",
+  "techreborn.message.coordsHaveBeen": "坐标以设置为",

```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "techreborn.message.coordsHaveBeen": "坐标已设置为",

```